packages/fxa-admin-panel/src/components/AccountSearch/index.tsx

```diff
@@ -49,20 +49,82 @@ export const GET_ACCOUNT_BY_EMAIL = gql`
   }
 `;
 
+export const GET_EMAILS = gql`
+  query fetchAll($search: String!) {
+    getEmailsLike(search: $search) {
+      email
+    }
+  }
+`;
+
 export const AccountSearch = () => {
   const [inputValue, setInputValue] = useState<string>('');
   const [showResult, setShowResult] = useState<Boolean>(false);
+  const [suggestions, setSuggestion] = useState<Array<string>>([]);
+  const [text, setText] = useState<string>('');
```

Comment:
What is `text`? Try to name things so they describe what they're used for. This is the value of the search input, right?